Documentation/Settings.cfg

```diff
@@ -1,41 +1,16 @@
 # coding: utf-8
 
-# #####
```

Comment:
Changes like these are out of Scope of the PR

Reply:
A separate commit should be enough, not?

Reply:
No, because the PR will be squash merged

Reply:
Yes. If you like having a squash merge, there will be a PR number referencing to this PR with both commits. And if you like to have cleaning ups separated from the task even in the git log, then let us merge this PR by simply merging it as-is.